docker/nwm_gui/web_server/nginx/default.conf

```diff
@@ -1,5 +1,6 @@
 upstream wresgui {
-    server nwm_gui_app_server:8000;
+    #server nwm_gui_app_server:8000;
```

Comment:
This can be removed since we know that there won't be a server at nwm_gui_app_server to serve as an upstream

Reply:
Comments removed as suggested.